website/docs/sdk-reference/community/vue.md

```diff
@@ -0,0 +1,252 @@
+```
+
+Create the logger with a specified log level:
+
+> Documentation: <https://configcat.com/docs/sdk-reference/js/#setting-log-levels>
+
+Use the logger in **clientOptions**:
+
+```js
+app.use(ConfigCatPlugin, {
+  sdkKey: "YOUR-CONFIGCAT-SDK-KEY-GOES-HERE", // // sdkKey is required
+  clientOptions: { // clientOptions is optional
+    // ...
+    logger: createConsoleLogger(LogLevel.Info),
+  }
+});
+```
+
+The following methods are available on **LogLevel**:
+
+- **LogLevel.Debug** - All events are logged.
+- **LogLevel.Info** - Info, Warn and Error are logged. Debug events are discarded.
+- **LogLevel.Warn** - Warn and Error events are logged. Info and Debug events are discarded.
+- **LogLevel.Error** - Error events are logged. All other events are discarded.
+- **LogLevel.Off** - No events are logged.
+
+### Specifying a User Object
+
+The [User Object](https://configcat.com/docs/advanced/user-object/) represents a user in your application. It works hand in hand with ConfigCat's [Targeting](https://configcat.com/docs/advanced/targeting/) rules for targeting specific users with feature flags. A User Object can be passed as a prop to the **FeatureWrapper** component.
+
+> See documentation here: <https://configcat.com/docs/advanced/user-object/>
+
+Define the User Object:
+
+```js
+<script setup lang="ts">
+import { ref } from 'vue';
+import { User } from 'configcat-vue';
+
+// Define the User Object
+const user = new User('john@example.com');
+const userObject = ref<User>(user)
+
+</script>
+```
+
+Pass it to the **FeatureWrapper** component:
+
+```js
+<template>
+  <div class="my-app">
+    <FeatureWrapper featureKey="YOUR-FEATURE-KEY-GOES-HERE" :userObject="userObject">
+      <TheNewFeature />
+      
+    </FeatureWrapper>
+  </div>
+</template>
+```
+
+### Listening to flag changes emitted from the FeatureWrapper component
+
+When a feature flag is toggled ON or OFF in the [ConfigCat dashboard](https://app.configcat.com) the **FeatureWrapper** component emits the updated feature flag value. How quickly the updated value is emitted depends on the polling interval set in the **clientOptions** property of the **ConfigCatPlugin**.
+
+Listen and handle changes using **@flag-value-changed**:
+
+```js
+<template>
+  <div class="my-app">
+    <FeatureWrapper featureKey="YOUR-FEATURE-KEY-GOES-HERE" @flag-value-changed="handleFlagValueChange">
+      <TheNewFeature />
+      
+    </FeatureWrapper>
+  </div>
+</template>
+```
+
+```js
+<script setup lang="ts">
+{/* Handle to the flag value changes */}
+const handleFlagValueChange = (flagValue: boolean) => {
+  console.log('Flag value changed to: ', flagValue);
+}
+
+</script>
+```
+
+### Listening to events emitted by the underlying ConfigCat client
+
+This plugin exposes (provides) the underlying ConfigCat client. One of the ways it can be used is by subscribing to events emitted by the ConfigCat client.
+
+> See documentation here: <https://configcat.com/docs/sdk-reference/js/#hooks>
```

Comment:
nit: The title could be more generic as using the client you can do much more than listening to events.

```suggestion
### Using the underlying ConfigCat client

This plugin exposes (provides) the underlying ConfigCat client. By injecting the provided client instance, you can use all the features it offers.

> See documentation here: <https://configcat.com/docs/sdk-reference/js>

One of the ways it can be used is by subscribing to events emitted by the ConfigCat client.
```